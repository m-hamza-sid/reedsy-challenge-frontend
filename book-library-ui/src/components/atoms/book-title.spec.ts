import { mount } from "@vue/test-utils";
import { describe, expect, test } from "vitest";
import BookTitle from "./book-title.vue";

describe("book-title", () => {
  test("should render with title prop", () => {
    const iTitle = "some title";
    const wrapper = mount(BookTitle, { props: { title: iTitle } });
    expect(wrapper.text()).toContain(iTitle);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
