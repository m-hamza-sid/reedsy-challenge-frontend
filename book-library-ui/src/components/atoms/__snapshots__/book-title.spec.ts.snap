// Vitest Snapshot v1

exports[`author > should render with name prop 1`] = `"<div class=\\"text-left\\">some name</div>"`;

exports[`book-title > should render with title prop 1`] = `"<h2 class=\\"text-2xl text-left font-bold\\">some title</h2>"`;
